vendor_skins/Evercast/libobs/obs.h

```diff
@@ -36,6 +36,8 @@
 #include "obs-properties.h"
 #include "obs-interaction.h"
 
+#define EVERCAST
```

Comment:
oh I see, we replace the header file and that'll be defined then. Can't we simply replace both header and impl files?

Reply:
Yeah, I'm not terribly happy with this entire set of changes.  We need to add this single parameter, but unfortunately, it's being added to an interface, which means dirtying a whole graph of code for this one little boolean.  The reason I didn't follow our usual practice with obs.c and copy it to vendor_skins is that it's is a pretty fundamental file.  If changes happen here upstream, they are very likely to be important, we really really don't want to miss out on them.  So, for this one instance, where we are traipsing through the file on our way somewhere else, I picked the `#ifdef` as the lesser of two evils.